src/NUnitFramework/framework/Constraints/EqualConstraint.cs

```diff
@@ -471,6 +491,9 @@ public override string Description
                 if (_comparer.IgnoreWhiteSpace)
                     sb.Append(", ignoring white-space");
 
+                if (_comparer.IgnoreWhiteSpace)
```

Comment:
I guess I'll have to fix this copypasta. I can do this after the first round of reviews.
```suggestion
                if (_comparer.NormalizeLineEndings)
```

Reply:
It does indicate that the change is not covered in nunit tests.
But neither are the original modifiers so it seems.